editor/src/clj/editor/fxui.clj

```diff
@@ -310,6 +347,7 @@
     (future
       (error-reporting/catch-all!
         (let [result @result-promise]
+          (fx/unmount-renderer state-atom renderer)
```

Comment:
Does this need to be called inside the `on-fx-thread` block, since it calls the renderer with `nil`?

Reply:
No, `renderer`, unlike JavaFX, is thread-safe: it already calls `on-fx-thread`.